fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -210,10 +217,11 @@ def parse_interfaces(self, conf, etree_conf):
             operation = resolve_operation(interface_node)
             if operation == "delete" and isinstance(port, AggregatedPort):
                 conf.remove_port(port)
+            if operation == "delete" and not isinstance(port, AggregatedPort) and not isinstance(port, VlanPort):
```

Comment:
if the juniper switch was currently handling vlan ports, the expected behavior would be to remove the vlan port and not reset it... now since we don't have the capability to add it i can't write a test for it, so i figured : let's save some headaches to the person implementing the vlan ports... i can remove that case and leave a comment, would you prefer i do that?